components/inputs/input-date-time-range-to.js

```diff
@@ -0,0 +1,198 @@
+import './input-text.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { bodySmallStyles } from '../typography/styles.js';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { LocalizeCoreElement } from '../../lang/localize-core-element.js';
+import ResizeObserver from 'resize-observer-polyfill/dist/ResizeObserver.es.js';
+import { SkeletonMixin } from '../skeleton/skeleton-mixin.js';
+
+class InputDateTimeRangeTo extends SkeletonMixin(LocalizeCoreElement(LitElement)) {
+
+	static get properties() {
+		return {
+			/**
+			 * Force block (stacked) range display if true
+			 */
+			blockDisplay: { attribute: 'block-display', type: Boolean },
```

Comment:
I don't actually see this being set -- is it meant to be used in a use case not yet captured?

Reply:
Ah I see it's used in the date-time-range PR 👍